### ISSUE TITLE
const totalHits = (this.hits[key] ?? 0) + 1| SyntaxError: Unexpected token '?'

### ISSUE THREAD

**Issue by i33mr** [nfriedly/express-rate-limit#274]
### Description

Hey, I just installed the new version (6.0.2). I faced to issues:

1. ESlint is giving me "Unable to resolve path to module 'express-rate-limit'" for some reason
2. When I run the code I'm getting this error: 
  ```
  node_modules\express-rate-limit\dist\index.cjs:49  
  const totalHits = (this.hits[key] ?? 0) + 1; 
  SyntaxError: Unexpected token '?'
```

Anyways, I uninstalled the version and used an older one (5.5.1), which works fine 

here is my code:

```js
const rateLimit = require("express-rate-limit");
const limiter = rateLimit({
  max: 100,
  windowMs: 60 * 60 * 1000,
  message: "Too many requests from this IP, please try again in an hour",
});
```

### Library version

6.0.2

### Node version

v12.18.4

### Module system

CommonJS

**Comment by nfriedly** [comment]
Oof, it looks like we need to bump the minimum node.js version, the nullish coalescing operator (`??`) doesn't work before 14.x - https://node.green/#ES2020-features--nullish-coalescing-operator-----

**Comment by nfriedly** [comment]
Ok, apologies for that, 6.0.3 now has a correct engines.node property to specify the minimum node.js version.

For your case, sticking with 5.x is good until you're ready to update to a newer node.js version.

**Comment by gamemaker1** [comment]
Hi @i33mr,

> ESlint is giving me "Unable to resolve path to module 'express-rate-limit'" for some reason

Are you still facing this issue? If yes, please watch #277 so any solution we reach there will help you too :)

**Comment by i33mr** [comment]
Thanks for the heads up @gamemaker1 !,

I'm currently using the 5.5.1 version which doesn't have that bug, I'll check it out when I consider updating.

**Comment by gamemaker1** [comment]
Hi @i33mr,

Quite a few others were facing the same issue even on Node 14, so we just released v6.3.0 with support for Node 12.9+. 

Could you please update with `npm install express-rate-limit@latest` and let us know if this fixes the issue?

Thanks!
